slides/lesson1/solution/main.tf

```diff
@@ -40,29 +40,35 @@ resource "aws_security_group" "web" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
-  tags = "${var.tags}"
+  tags = var.tags
 }
 
 resource "aws_instance" "web" {
-  ami           = "${data.aws_ami.amazon_linux.id}"
+  ami           = data.aws_ami.amazon_linux.id
   instance_type = "t2.micro"
   key_name      = "academy-ohio"
```

Comment:
Being that this depends on the user already having this key pair, how about we add this as a variable?

Reply:
I added a variable on `variables.tf` for them to include.